### PR TITLE
gtk: Generate Container::[get|set]_propert_child

### LINKED ISSUE #131: missing set|get_child_property on GtkContainer
I'm not sure if this are safe to generate ?

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #131.

It was mentioned in #131 that we needed to be careful how we would expose this API in rust. Is there anything special to do?

### CODE REVIEW

File: gtk/src/auto/container.rs [hunk 1]
```diff
@@ -175,6 +186,19 @@ impl<O: IsA<Container>> ContainerExt for O {
     //    unsafe { TODO: call ffi:gtk_container_child_get() }
     //}
 
+    fn child_get_property(&self, child: &impl IsA<Widget>, property_name: &str) -> glib::Value {
```
Comment (bilelmoussaoui) [inline gtk/src/auto/container.rs:189]: Maybe we can make the gir produces code that make use of this once merged instead of going through ffi as well

Comment (sdroege) [reply gtk/src/auto/container.rs:189]: Yes please. Just like your PR for normal properties :)

File: gtk/src/auto/container.rs [hunk 2]
```diff
@@ -203,6 +227,22 @@ impl<O: IsA<Container>> ContainerExt for O {
...
     //}
 
+    fn child_set_property(
+        &self,
+        child: &impl IsA<Widget>,
+        property_name: &str,
+        value: &glib::Value,
+    ) {
+        unsafe {
+            ffi::gtk_container_child_set_property(
```
Comment (sdroege) [inline gtk/src/auto/container.rs:237]: This needs checking if the property exists, if the value is of the correct type, if the property is writable

File: gtk/src/auto/container.rs [hunk 3]
```diff
@@ -175,6 +186,19 @@ impl<O: IsA<Container>> ContainerExt for O {
     //    unsafe { TODO: call ffi:gtk_container_child_get() }
     //}
 
+    fn child_get_property(&self, child: &impl IsA<Widget>, property_name: &str) -> glib::Value {
+        unsafe {
+            let mut value = glib::Value::uninitialized();
+            ffi::gtk_container_child_get_property(
```
Comment (sdroege) [inline gtk/src/auto/container.rs:192]: This needs checking if the property exists and is readable. You return garbage from here if that's not the case :)

Comment (bilelmoussaoui) [reply gtk/src/auto/container.rs:192]: This basically needs something like there's in ObjectExt

**Pull request closed without merging**